core/src/mindustry/core/NetServer.java

```diff
@@ -422,6 +426,8 @@ boolean checkPass(){
                     builder.append("[lightgray] ").append(p.name).append("[accent] (#").append(p.id()).append(")\n");
                 });
                 player.sendMessage(builder.toString());
+            }else if(args.length == 1){
+                player.sendMessage("[orange]You need a valid reason to kick the player. Add a reason after the player name.");
```

Comment:
This would break the votekick button in the player list.

Reply:
Fixed that.